generators/app/templates/custom/admin/_tracker.html

```diff
@@ -0,0 +1,17 @@
+<ion-view view-title="{{'tracker.title'| translate}}">
```

Comment:
Make this line look like [this](https://github.com/ruddell/generator-jhipster-ionic/blob/fa3f9b49fad00f3ed0dfeb89a49738651ec7b1fa/generators/app/templates/custom/account/_settings.html#L1).  The translate regex doesn't pick this up currently